app/templates/components/event-card.hbs

```diff
@@ -1,28 +1,61 @@
-<div class="ui fluid event card">
```

Comment:
@niranjan94

The `divs` having common code have different scopes for closing.  Even if we write it using if conditions, the complexity will just increase. will have to rewrite most of the code. What do you say?